### PR TITLE
Nav links and dropdowns

### LINKED ISSUE #127: Refactor links
- add dropdown for user that has user info and orders
 - add dropdown for admin for admin dashboard, all items, user info, orders.
- add all products on products dropdown

### PR DESCRIPTION
**Pull request by CjMoore**
- adds dropdown for admin user 
- adds dropdown for default user

closes #127

### CODE REVIEW

File: app/views/shared/_nav.html.erb [hunk 1]
```diff
@@ -1,16 +1,41 @@
...
+    <li><%= link_to "Orders", orders_path %></li>
+    <li class="divider"></li>
+</ul>
+
+<ul id="dropdown3" class="dropdown-content user-dropdown">
+    <li><%= link_to "User Info", dashboard_path %></li>
+    <li class="divider"></li>
+    <li><%= link_to "Orders", orders_path %></li>
+    <li class="divider"></li>
+</ul>
```
Comment (meyerhoferc) [inline app/views/shared/_nav.html.erb:26]: going to be so much cleaner with this change

File: app/views/shared/_nav.html.erb [hunk 2]
```diff
@@ -1,16 +1,41 @@
...
    <div class="nav-wrapper orange darken-3">
      <div class ="container">
        <a href="<%= root_path %>" class="brand-logo">Weasleys' Wizard Wheezes</a>
        <ul id="nav-mobile" class="right hide-on-med-and-down">
          <li><a class="dropdown-button" href="/" data-activates="dropdown1">Products</a></li>
+          <% if current_user && current_admin? %>
+            <li><a class="dropdown-button" href="/" data-activates="dropdown2">Admin Tools</a></li>
+          <% elsif current_user %>
+            <li><a class="dropdown-button" href="/" data-activates="dropdown3">User Tools</a></li>
+          <% end %>
```
Comment (meyerhoferc) [inline app/views/shared/_nav.html.erb:38]: 👍

File: spec/features/nav_footer/user_sees_nav_and_footer_spec.rb [hunk 3]
```diff
@@ -47,51 +47,67 @@
...
-    within('.nav-wrapper') do
-      expect(page).to have_link("Logout")
+    within("nav") do
+      expect(page).to_not have_link("Admin Tools")
+      expect(page).to have_link("User Tools")
+    end
+
+    within(".user-dropdown") do
+      expect(page).to have_link("User Info")
+      expect(page).to have_link("Orders")
```
Comment (meyerhoferc) [inline spec/features/nav_footer/user_sees_nav_and_footer_spec.rb:110]: 👍

**Review by meyerhoferc — APPROVED** [review]
Looks great, I think this will really help our UI. Ready for second review & merge @valerietrudell @cmacaulay

File: app/views/shared/_nav.html.erb (hunk 1 above)
Comment (valerietrudell) [inline app/views/shared/_nav.html.erb]: :1st_place_medal:

**Comment by valerietrudell** [comment]
Looks good I'm going to merge

**Pull request merged**